crates/turborepo-lib/src/config/mod.rs

```diff
@@ -329,6 +330,15 @@ impl ConfigurationOptions {
     }
 
     pub fn daemon(&self) -> Option<bool> {
+        // hardcode to off in CI
+        if turborepo_ci::is_ci() {
+            if Some(true) == self.daemon {
+                warn!("Ignoring daemon setting and disabling the daemon because we're in CI");
```

Comment:
I think we might want to make this debug instead. If you have `daemon: true` in your config, you will ALWAYS get this warning in CI with no way to turn it off.